app/cafe.py

```diff
@@ -0,0 +1,23 @@
+from datetime import date
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("All friends should be vaccinated")
+
+        if date.today() > visitor["vaccine"]["expiration_date"]:
+            raise OutdatedVaccineError("All friends should be vaccinated")
+
+        if visitor["wearing_a_mask"] is False:
+            raise NotWearingMaskError("All visitors must wear masks")
```

Comment:
add descriptive error message; an error message is strange. You perform checks for one visitor, so if he is not vaccinated why the message is "all friends". Who are these friends?
I mean use visitor name for better error message : )